operators/pkg/forge/cloudinit.go

```diff
@@ -75,18 +74,13 @@ func CloudInitUserData(nextcloudBaseURL, webdavUsername, webdavPassword string,
 			ID0:     interf{DHCP4: true},
 		},
 		Mounts: [][]string{{
-			fmt.Sprintf("%s/remote.php/dav/files/%s", nextcloudBaseURL, webdavUsername),
-			"/media/MyDrive",
-			"davfs",
-			"_netdev,auto,user,rw,uid=1000,gid=1000",
+			fmt.Sprintf("%s:%s", serviceName, servicePath),
+			"/media/nfs-share",
```

Comment:
Would revert the mountpoint: iirc, basically all images have a shortcut pointing to there...

Reply:
Reverted mount point